spec/features/wcs_spec.rb

```diff
@@ -1,32 +1,28 @@
-describe 'Homepage', type: :feature do
-
-  before do
-    Capybara.app_host = 'http://localhost:3000'
-  end
+require 'rails_helper'
 
+describe 'Homepage', type: :feature do
   it 'should load the home page' do
     visit '/'
     expect(page).to have_text('West Coast Skateparks')
   end
 
+  # moved this from search spec because it seems more relevant here
+  it 'should have working links' do
+    visit '/'
+    click_link('Skateparks')
+    expect(page).to have_text('California')
+  end
 end
 
 describe 'Show', type: :feature do
-
   it 'displays skatepark information' do
     visit '/skateparks/269'
-    expect(page).to have_text('Ojai')
-    expect(page).to have_text('Site Design')
+    expect(page).to have_text('Oceanside')
+    expect(page).to have_text('Martin Luther King Jr Skatepark')
   end
```

Comment:
This spec might fail for you because our development data might be slightly different, just a heads up.